docs/src/tutorials/basic/sample.rst

```diff
@@ -1,4 +1,4 @@
-Sample
+Basic
 ======
 
 The :py:mod:`easyreflectometry` package is focused on making easy to use functionality for specific modelling approaches. 
```

Comment:
I'm not sure that I follow what either "Sample" or "Basic" refers to here. Looking at the current docs, it seems to me that this part of the package covers the fundamental building blocks for creating a model of a sample. Maybe one could call this section "Building a structure" or "Creating a sample"?

Reply:
Ah I see now that you've restructured the entire docs compared with the currently published ones. The name makes more sense now that is under tutorials.